app/views/course/statistics/assessments/ancestor_statistics.json.jbuilder

```diff
@@ -8,20 +8,21 @@ json.assessment do
   json.url course_assessment_path(current_course, @assessment)
 end
 
-json.submissions @submission_records do |record|
+json.submissions @student_submissions_hash.each do |course_user, (submission, end_at)|
   json.courseUser do
-    json.id record[0].id
-    json.name record[0].name
-    json.role record[0].role
-    json.isPhantom record[0].phantom?
+    json.id course_user.id
+    json.name course_user.name
+    json.role course_user.role
+    json.isPhantom course_user.phantom?
   end
 
-  json.submittedAt record[1]&.iso8601
-  json.endAt record[2]&.iso8601
-  json.grade record[3]
+  json.workflowState submission.workflow_state
+  json.submittedAt submission.submitted_at&.iso8601
+  json.endAt end_at&.iso8601
+  json.totalGrade submission.grade
 end
 
-json.allStudents @all_students do |student|
+json.allStudents @all_students.each do |student|
```

Comment:
there's no difference actually between using `each` and not using it; both will iterate through all the items inside `@all_students`. The reason I change this into using `each` is so that it will be consistent with other similar code snippet that I made within this PR

also, using `each` makes the code put an emphasis (and hence, in my opinion, more understandable by the reader) more in actually iterating through each one of the members, as opposed to not using it..